security-jwt/src/test/groovy/io/micronaut/security/token/jwt/cookie/JwtCookieConfigurationSpec.groovy

```diff
@@ -10,6 +10,10 @@ class JwtCookieConfigurationSpec extends ApplicationContextSpecification {
     @Shared
     JwtCookieConfiguration jwtCookieConfiguration = applicationContext.getBean(JwtCookieConfiguration)
 
+    Map<String, Object> getConfiguration() {
+        ['micronaut.security.authentication': 'cookie']
```

Comment:
`super.configuration + ['micronaut.security.authentication': 'cookie']`

instead of 

` ['micronaut.security.authentication': 'cookie']`